static/app/components/replays/table/replayTableColumns.tsx

```diff
@@ -522,18 +523,40 @@ export const ReplaySessionColumn: ReplayTableColumn = {
 
     const referrer = getRouteStringFromRoutes(routes);
     const eventView = EventView.fromLocation(location);
+
+    const {statsPeriod, ...eventViewQuery} = eventView.generateQueryStringObject();
+
+    let query = {
+      referrer,
+      ...eventViewQuery,
+    };
+
+    if (!eventViewQuery.start && !eventViewQuery.end && typeof statsPeriod === 'string') {
```

Comment:
This only checks for `statsPeriod`, now that we translate to `playlistStart`/`End`, we need to also account for `start` and `end`